modules/swagger-jaxrs2/src/main/java/io/swagger/jaxrs2/OperationParser.java

```diff
@@ -0,0 +1,231 @@
+package io.swagger.jaxrs2;
+
+import io.swagger.converter.ModelConverters;
+import io.swagger.oas.annotations.media.ExampleObject;
+import io.swagger.oas.models.ExternalDocumentation;
+import io.swagger.oas.models.examples.Example;
+import io.swagger.oas.models.info.Contact;
+import io.swagger.oas.models.info.Info;
+import io.swagger.oas.models.info.License;
+import io.swagger.oas.models.media.Content;
+import io.swagger.oas.models.media.MediaType;
+import io.swagger.oas.models.media.Schema;
+import io.swagger.oas.models.parameters.Parameter;
+import io.swagger.oas.models.parameters.RequestBody;
+import io.swagger.oas.models.responses.ApiResponse;
+import io.swagger.oas.models.responses.ApiResponses;
+import io.swagger.oas.models.servers.Server;
+import io.swagger.oas.models.servers.ServerVariable;
+import io.swagger.oas.models.servers.ServerVariables;
+import io.swagger.oas.models.tags.Tag;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+/**
+ * Created by RafaelLopez on 5/27/17.
+ */
+public class OperationParser {
+
+	public static Optional<List<Parameter>> getParametersList(io.swagger.oas.annotations.Parameter[] parameters) {
+		if (parameters == null) {
+			return Optional.empty();
+		}
+		List<Parameter> parametersObject = new ArrayList<>();
+		for (io.swagger.oas.annotations.Parameter parameter : parameters) {
+			Parameter parameterObject = new Parameter();
```

Comment:
I think this logic isn't right.  Parameters may be _explicit_ by being decorated with `@Schema` annotations, or with `@QueryParam`, etc.  Look at the `ParameterProcessor` logic from the previous version for more details on this.